Review markup and fix images (#36)
sora-xor/sora-docs: src/advanced-trading.md

<center><img src="/.gitbook/assets/advanced-trade-tbc-xor-price.png" width="400"></center>

The price for XOR when the **SMART** option is selected:

<center><img src="/.gitbook/assets/advanced-trade-smart-xor-price.png" width="400"></center>

The price of XOR differs depending on whether you are buying from TBC or from XYK Pool.

Let's go through exchange parameters one by one:

**TBC parameters:**

* *VAL PER XOR*: The price of 1 XOR in VAL.
* *XOR PER VAL*: The price of 1 VAL in XOR.
* *Minimum Received*: The minimum number of destination assets that you'll get after the swap
* *Liquidity provider fee*: The fee that liquidity provider will take. In this case, it's a TBC.
* _Network fee_: SORA fee for processing the transaction (gas).

**XYK Pool parameters:**

* *VAL PER XOR*: The price of 1 XOR in VAL.
* *XOR PER VAL*: The price of 1 VAL in XOR.
* *Minimum Received*: The minimum number of destination assets that you'll get after the swap.
* *Liquidity provider fee*: The fee that liquidity provider will take. In this case, it's an XYK Pool.
* *Network fee*: SORA fee for processing the transaction (gas).

#### What Insufficient _liquidity_ means

> You can get the error saying "**Not enough liquidity**". It's important to understand what this means.

Reviewer: ```suggestion
It is possible to get an error saying "**Not enough liquidity**" and understanding what this means is important.
```
